tests/integration/specs/utils.js

```diff
@@ -185,7 +185,7 @@ export function validateEntity(dataStoreType, collectionName, expectedEntity, se
 export function cleanUpCollectionData(collectionName) {
   const networkStore = Kinvey.DataStore.collection(collectionName, Kinvey.DataStoreType.Network);
   const syncStore = Kinvey.DataStore.collection(collectionName, Kinvey.DataStoreType.Sync);
-  return networkStore.find().toPromise()
+  return networkStore.find()
```

Comment:
Does find now return an object for all stores and would code expecting observable still work?

Reply:
Initially we were going to remove the observable api but decided against this breaking change. Instead the `Auto` datastore type won't use observables since it a new api. This change has been reverted.